src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsEvolveObject.java

```diff
@@ -10,7 +10,7 @@
 
 public class AdsEvolveObject extends AdsBaseObject {
 
-  private static final String EVOLVE_SELECTOR = " script[src*=\"4403ad\"]";
+  private static final String EVOLVE_SELECTOR = " [id*='wikia_gpt/4403/ev/wikia_intl']";
```

Comment:
It's up to you but I prefer to see what the type of element from the first glance and this thing is under our control.